consensus/src/pow/work.rs

```diff
@@ -255,7 +254,7 @@ pub fn mine(
             return Ok(MiningResult::Success);
         }
 
-        if stop_flag.load(std::sync::atomic::Ordering::Relaxed) {
+        if stop_flag.load(std::sync::atomic::Ordering::SeqCst) {
```

Comment:
This can be relaxed. In fact, this is an excellent example for using relaxed memory order. The question is: What's the worst thing that will happen if this is relaxed? The answer is: It'll be delayed a few cycles and stop eventually? Who cares.